.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The specific settings that you're using in your specific editor should not be committed to the repository.